project/Build.scala

```diff
@@ -891,7 +891,14 @@ object Build extends sbt.Build {
           case FullOptStage => "-tfullopt-stage"
         })
 
-        envTags ++ semTags :+ stageTag
+        val strongModeTags = (scalaJSOutputMode in Test).value match {
```

Comment:
Consider calling this simply `modeTag`, its not like there will be multiple tags for strong mode, but we might have other tags that are mode specific.